src/sentry/grouping/enhancer/matchers.py

```diff
@@ -269,6 +273,8 @@ class CategoryMatch(FrameFieldMatch):
 
 
 class ExceptionFieldMatch(FrameMatch):
+    field_path: Any = None
```

Comment:
I think this should be `field_path: list[str]` (without the `= None`)

really it should be `tuple[str, ...]` but then you'd have to fix all the subclasses to use tuples instead of lists which might be out of scope